lib/prisma.ts

```diff
@@ -19,8 +19,20 @@ const POSTGRESQL_DATE_FORMATS = {
   year: 'YYYY-01-01',
 };
 
-function getAddMinutesQuery(field: string, minutes: number) {
-  const db = getDatabaseType();
+function toUuid(): string {
```

Comment:
This function is removed. The rawQuery handles this internally. You can just do {{var::uuid}}.

Reply:
removed uuid function